docs/validators/mainnet.md

```diff
@@ -20,7 +20,7 @@ You need to set the **genesis file** and **seeds**. If you need more information
 
 | Chain ID       | Description     | Site                                                               | Version                                                      | Status  |
 | -------------- | --------------- | ------------------------------------------------------------------ | ------------------------------------------------------------ | ------- |
-| `evmos_9001-2` | Evmos Mainnet 2 | [Evmos](https://github.com/tharsis/mainnet/tree/main/evmos_9001-2) | [`{{ $themeConfig.project.latest_version }}`](https://github.com/evmos/evmos/releases) | `Live`  |
```

Comment:
@cryptobtcbuyer Instead of hardcoding the version, I suggest adding a note explaining that the current Software version linked might differ if an active Software Upgrade proposal exists.

Reply:
Yes, I agree with you.

I made this change because I ran in DM into a lot of questions about the "panic: UnmarshalJSON cannot decode empty bytes" error . People were guided by the version in the documentation and installed the wrong release. It will be great if you add this note

Reply:
@cryptobtcbuyer feel free to add the note and we can merge